cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -247,4 +257,15 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
             outputDir.resolve(relativePath.toString()).safeDeleteRecursively()
         }
     }
+
+    private fun archiveAll() {
+        val zipFile = createTempFile(ORT_NAME, ".zip").toFile()
+        zipFile.deleteOnExit()
+
+        outputDir.packZip(zipFile, overwrite = true)
+
+        outputDir.deleteRecursively()
+
+        zipFile.copyTo(outputDir.resolve("archive.zip"))
```

Comment:
Why don't you create the ZIP in the final location in the first place?

Reply:
If you create the ZIP in the download directory and then try to pack everything in the directory, an infinite loop is created. It tries to pack itself and so on. However, I think the `pack` method allows an additional filter method, so this could be done with that.

Reply:
I decided to use the filter method, it reduces the code a lot.